src/main/java/com/github/tennaito/rsql/misc/DefaultArgumentParser.java

```diff
@@ -87,6 +88,9 @@
         // 1.8 date
 		if (type.equals(LocalDate.class)) {
 			return (T) LocalDate.parse(argument);
+        }
```

Comment:
For the sake of similarity, could you please replace your four spaces indentation with tabs?